src/test/java/seedu/address/storage/JsonSerializableAddressBookTest.java

```diff
@@ -20,23 +20,24 @@
     private static final Path INVALID_PERSON_FILE = TEST_DATA_FOLDER.resolve("invalidPersonAddressBook.json");
     private static final Path DUPLICATE_PERSON_FILE = TEST_DATA_FOLDER.resolve("duplicatePersonAddressBook.json");
 
-    @Test
+    @BeforeEach
     public void toModelType_typicalPersonsFile_success() throws Exception {
         JsonSerializableAddressBook dataFromFile = JsonUtil.readJsonFile(TYPICAL_PERSONS_FILE,
                 JsonSerializableAddressBook.class).get();
         AddressBook addressBookFromFile = dataFromFile.toModelType();
+        System.out.println(dataFromFile);
```

Comment:
I think this line should not be in this test right?

Reply:
yes, this is the rough version, update it in half an hour